app/policies/service_provider_mfa_policy.rb

```diff
@@ -25,27 +27,29 @@ def user_needs_sp_auth_method_verification?
 
     if piv_cac_required?
       auth_method.to_s != 'piv_cac'
-    elsif aal3_required?
-      !AAL3_METHODS.include?(auth_method.to_s)
+    elsif phishing_resistant_required?
+      !PHISHING_RESISTANT_METHODS.include?(auth_method.to_s)
     else
       false
     end
   end
 
   def user_needs_sp_auth_method_setup?
     return true if piv_cac_required? && !piv_cac_enabled?
-    return true if aal3_required? && !aal3_enabled?
+    return true if phishing_resistant_required? && !phishing_resistant_enabled?
     false
   end
 
   def auth_method_confirms_to_sp_request?
     !user_needs_sp_auth_method_setup? && !user_needs_sp_auth_method_verification?
   end
 
-  def aal3_required?
-    return aal3_requested? if aal_requested?
-
-    service_provider&.default_aal == 3
+  def phishing_resistant_required?
+    if phishing_resistant_requested? || aal_requested?
+      !!phishing_resistant_requested?
+    else
+      service_provider&.default_aal == 3
```

Comment:
**comment:** I'm good to still use `3` as the `default_aal` value but we may ultimately want to rename the column so we can include `hspd12` and use an enum with more descriptive values.

Reply:
https://cm-jira.usa.gov/browse/LG-7722 for posterity 😄